feature/experimental/p4rt/ate_tests/performance_test/performance_test.go

```diff
@@ -0,0 +1,932 @@
+		payload = append(payload, byte(i))
+	}
+	gopacket.SerializeLayers(buf, opts,
+		pktEth, gopacket.Payload(payload),
+	)
+	return buf.Bytes()
+}
+
+// packetTracerouteRequestGet generates PacketOut payload for Traceroute packets.
+func packetTracerouteRequestGet(srcMAC, dstMAC net.HardwareAddr, isIPv4 bool, ttl uint8, seq int) ([]byte, error) {
+	buf := gopacket.NewSerializeBuffer()
+	opts := gopacket.SerializeOptions{
+		FixLengths:       true,
+		ComputeChecksums: true,
+	}
+	var payload []byte
+	payLoadLen := 32
+
+	ethType := layers.EthernetTypeIPv4
+	if !isIPv4 {
+		ethType = layers.EthernetTypeIPv6
+	}
+	pktEth := &layers.Ethernet{
+		SrcMAC:       srcMAC,
+		DstMAC:       dstMAC,
+		EthernetType: ethType,
+	}
+
+	pktIpv4 := &layers.IPv4{
+		Version:  4,
+		TTL:      ttl,
+		SrcIP:    net.ParseIP(dutPort1.IPv4).To4(),
+		DstIP:    net.ParseIP(atePort1.IPv4).To4(),
+		Protocol: layers.IPProtocolICMPv4,
+		Flags:    layers.IPv4DontFragment,
+	}
+	pktICMP4 := &layers.ICMPv4{
+		TypeCode: layers.CreateICMPv4TypeCode(layers.ICMPv4TypeEchoRequest, 0),
+		Seq:      uint16(seq),
+	}
+
+	pktIpv6 := &layers.IPv6{
+		Version:    6,
+		HopLimit:   ttl,
+		NextHeader: layers.IPProtocolICMPv6,
+		SrcIP:      net.ParseIP(dutPort1.IPv6).To16(),
+		DstIP:      net.ParseIP(atePort1.IPv6).To16(),
+	}
+	pktICMP6 := &layers.ICMPv6{
+		TypeCode: layers.CreateICMPv6TypeCode(layers.ICMPv6TypeEchoRequest, 0),
+	}
+	pktICMP6.SetNetworkLayerForChecksum(pktIpv6)
+
+	for i := 0; i < payLoadLen; i++ {
+		payload = append(payload, byte(i))
+	}
+	if isIPv4 {
+		if err := gopacket.SerializeLayers(buf, opts,
+			pktEth, pktIpv4, pktICMP4, gopacket.Payload(payload),
+		); err != nil {
+			return nil, err
+		}
+		return buf.Bytes(), nil
+	}
+	if err := gopacket.SerializeLayers(buf, opts,
+		pktEth, pktIpv6, pktICMP6, gopacket.Payload(payload),
+	); err != nil {
+		return nil, err
+	}
+	return buf.Bytes(), nil
+}
+
+// packetLLDPRequestGet generates PacketOut payload for LLDP packets.
+func packetLLDPRequestGet() []byte {
+	buf := gopacket.NewSerializeBuffer()
+	opts := gopacket.SerializeOptions{
+		FixLengths:       true,
+		ComputeChecksums: true,
+	}
+	pktEth := &layers.Ethernet{
+		SrcMAC: net.HardwareAddr{0x00, 0xAA, 0x00, 0xAA, 0x00, 0xAA},
+		// LLDP MAC is 01:80:C2:00:00:0E
+		DstMAC:       net.HardwareAddr{0x01, 0x80, 0xC2, 0x00, 0x00, 0x0E},
+		EthernetType: lldpInLayers,
+	}
+
+	pktLLDP := &layers.LinkLayerDiscovery{
+		ChassisID: layers.LLDPChassisID{
+			Subtype: layers.LLDPChassisIDSubTypeMACAddr,
+			ID:      []byte{0x01, 0x01, 0x01, 0x01, 0x01, 0x01},
+		},
+		PortID: layers.LLDPPortID{
+			Subtype: layers.LLDPPortIDSubtypeIfaceName,
+			ID:      []byte("port1"),
+		},
+		TTL: 100,
+	}
+
+	gopacket.SerializeLayers(buf, opts, pktEth, pktLLDP)
+	return buf.Bytes()
+}
+
+// GetTableEntry creates wbb acl entry related to GDP,LLDP and traceroute.
+func getTableEntry(actionType p4_v1.Update_Type) []*p4rtutils.ACLWbbIngressTableEntryInfo {
```

Comment:
Go function names should not start with Get
https://google.github.io/styleguide/go/decisions#getters

Maybe these should e "newTableEntry", "newPacketOut", "newTrafficFlow"?